{
  "expire": null,
  "issue": {
    "source": "気象庁",
    "time": "2019/10/28 09:19:00",
    "type": "ScaleAndDestination",
    "correct": "None"
  },
  "earthquake": {
    "time": "2019/10/28 09:16:00",
    "hypocenter": {
      "name": "宮城県沖",
      "latitude": 38.2,
      "longitude": 141.9,
      "depth": 50,
      "magnitude": 4.0
    },
    "maxScale": 20,
    "domesticTsunami": "None",
    "foreignTsunami": "Unknown"
  },
  "points": []
}